docs/samples/advanced_config/aggregator.yaml

```
@@ -162,3 +162,31 @@ garbage_collection:
   # The maximum number of collection jobs (& related artifacts), per task, to delete in a single run
   # of the garbage collector.
   collection_limit: 50
+
+# Configuration for key rotator. Allows running the key rotator as part of the aggregator process.
+# If omitted, you should run the key rotator as a separate cronjob.
+key_rotator:
+  # How frequently to run the key rotator, in seconds. Required.
+  frequency_s: 3600
+
+  # Rotation policy for global HPKE keys.
+  hpke:
+    # How long keys remains pending before they're promoted to active. Should
+    # be greater than the global HPKE keypair cache refresh rate. Defaults to
+    # 1 hour.
+    pending_duration_s: 3600
+
+    # The TTL of keys. Defaults to 4 weeks.
```

```suggestion
    # The TTL of keys. Defaults to 12 weeks.
```

(nit, I believe this expresses the current state of the code)